### ISSUE TITLE
 First run - This content failed to load.

### ISSUE THREAD

**Issue by motatuc** [nextcloud/server#2248]
Hello.

Just installed Nextcloud on my server with the setup-nextcloud.php-script.
Finished without any problems.
Login working fine. But loading content is not possible - opening layers stay empty and show: This content failed to load.

### Steps to reproduce
1. Login Nextcloud
2. Layer window opens
3. Error message and window stays empty.
4. Choose another app, or any other option -> empty layer window with same error message.

### Expected behaviour
This window should contain content.

### Actual behaviour
Window stays empty and throws error message.

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.

-> No errors have been found.
```

**List of activated apps:**

Standard activated apps.

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**PHP-VERSION:** "PHP/5.5.38"

My browser also tells me this:
.../nextcloud/index.php/apps/firstrunwizard/wizard.php -> 404
../nextcloud/index.php/apps/files/ajax/getstoragestats.php?dir=/ -> 404
../nextcloud/index.php/apps/firstrunwizard/ajax/disable.php -> 404

I also see this:
Undefined index: MemAvailable at /nfs/xxx/www/webserver/htdocs/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85

![screenshot from 2016-11-22 17-17-23](https://cloud.githubusercontent.com/assets/15171154/20532996/471f5e5c-b0dc-11e6-8273-491538978d66.png)

**Comment by nickvergessen** [comment]
This will be fixed for 10.0.2
The patch is at https://github.com/nextcloud/serverinfo/pull/59/files in case you want to already patch it.

Alternatively you can also disable the serverinfo app

**Comment by jgallucci32** [comment]
I just installed 10.0.1 and ran your patch as well as disabled the serverinfo app. Still getting the same window poping up every 5 seconds as the page automatically refreshes.

I am running on RHEL7 with PhP 5.4.16 (latest) and Httpd 2.4.6 (latest) installed from RHEL repositories.
